ops/terraform/modules/resources/s3_pii/templates/bucket-policy.json

```diff
@@ -23,14 +23,13 @@ ${jsonencode(
       "Effect": "Allow",
       "Principal": {
         "AWS": [
-          for arn in concat(readers, writers) : "${arn}"
+          for arn in sort(concat(readers, writers)) : "${arn}"
```

Comment:
just curious....benefit(s) of sorting the ARN(s) ?

Reply:
> just curious....benefit(s) of sorting the ARN(s) ?

Excellent question.

There was some debate as to whether this solved _anything_ at all, in fact. After trial and error (read: threashing), ensuring a consistent order _seems_ to result in a more consistent diff when running terraform plan/apply. I'm hedging here, because it wasn't as controlled of an evaluation as we would have liked–dealing with a years old terraform binary and years old provider version had us looking at similarly aged issues on this matter. We updated the provider version somewhere along the way and what's more, we were mostly using partial plan/apply via the `-target` flag which is convenient, but it's not quite the same as allowing terraform to run through a full evaluation to produce a meaningful diff.